Packs/FireEyeETP/Integrations/FireEyeETP/FireEyeETP.py

```diff
@@ -478,6 +558,86 @@ def get_alerts_command():
     demisto.results(entry)
 
 
+def upload_yara_file_command(client, args):
+
+    entry_id = args.get('entryID')
+    policy_uuid = args.get('policy_uuid')
+    ruleset_uuid = args.get('ruleset_uuid')
+
+    file_obj = demisto.getFilePath(entry_id)
+    file_path = file_obj['path']
+
+    with open(file_obj.get('path'), "rb") as file:
+        data=file.read()
+        files = {
+            'file': ('new.yara', data)
+        }
+        response = client.upload_yara_file(policy_uuid, ruleset_uuid, files)
+        #print (response2)
+        if response.status_code == 202:
+            return "Upload of Yara file succesfully."
+        else :
+            return "Upload of Yara file failed."
+
+
+def get_events_data_command(client, args):
+
+    message_id = args.get('message_id')
+
+    response = client.get_events_data(message_id)
+
+
+    result_output = dict()
+    result_output['Logs'] = response['data'][message_id]
+
+    for log in result_output['Logs']:
+        if log['action_on_msg'] == "MTA_RCPT_DELIVERED_OUTBOUND":
+            result_output['Delivered_msg'] = log['display_msg']
```

Comment:
why do you use the same dictionary- all will be in the context data.

Reply:
because we have found it interesting to include as output the sending status of the email, the message and, if applicable, the Internet Message ID

Reply:
only the last log Delivered_msg, Delivered_status will appear- is that what you ment?

Reply:
Ok amazing got you!